### PR TITLE
Add mediators keybase usernames

### PR DESCRIPTION
**Pull request by chimp1984**

### CODE REVIEW

**Comment by chimp1984** [comment]
Screen Shot 2020-12-25 at 13 58 57
Screen Shot 2020-12-25 at 13 59 36
Screen Shot 2020-12-25 at 14 00 29

File: desktop/src/main/java/bisq/desktop/main/overlays/windows/TradeDetailsWindow.java [hunk 1]
```diff
@@ -358,6 +360,9 @@ private void addContent() {
                     data += "\n\nRaw deposit transaction as hex:\n" + depositTxAsHex;
                 }
 
+                data += "\n\nSelected mediator: " + DisputeAgentLookupMap.getKeyBaseUserName(contract.getMediatorNodeAddress().getFullAddress());
+                data += "\nSelected arbitrator (refund agent): " + DisputeAgentLookupMap.getKeyBaseUserName(contract.getRefundAgentNodeAddress().getFullAddress());
+
```
Comment (ghubstan) [inline desktop/src/main/java/bisq/desktop/main/overlays/windows/TradeDetailsWindow.java:365]: Some strings are prefixed with `\n\n`, others with `\n`.  If this is the intent I'm not suggesting a change.

**Review by ghubstan — APPROVED** [review]
utACK

File: desktop/src/main/java/bisq/desktop/main/overlays/windows/TradeDetailsWindow.java (hunk 1 above)
Comment (chimp1984) [reply desktop/src/main/java/bisq/desktop/main/overlays/windows/TradeDetailsWindow.java:365]: Yes, its intentional. Its an additional linebreak before the first line.

**Review by ripcurlx — APPROVED** [review]
utACK

**Pull request merged**